paths/accounts/locales.yaml

```diff
@@ -0,0 +1,42 @@
+---
+summary: List locales used in account
+description: List all locales used across all projects within an account.
+operationId: account/locales
+tags:
+- Locales
+parameters:
+- "$ref": "../../parameters.yaml#/X-PhraseApp-OTP"
+responses:
+  '200':
+    description: OK
+    content:
+      application/json:
+        schema:
+          type: array
+          items:
+            "$ref": "../../schemas/locale_preview.yaml#/locale_preview"
```

Comment:
@forelabs i also noticed, that this type will not match, if the suggestion from this PR will be added: https://github.com/phrase/phrase/pull/8044#pullrequestreview-682653475